packages/jellyfin-server/build.sh

```diff
@@ -0,0 +1,26 @@
+TERMUX_PKG_HOMEPAGE="https://jellyfin.org"
+TERMUX_PKG_DESCRIPTION="A free media system for organizing and streaming media"
+TERMUX_PKG_LICENSE="GPL-2.0"
+TERMUX_PKG_VERSION="10.10.5"
+TERMUX_PKG_SRCURL="https://github.com/jellyfin/jellyfin/archive/refs/tags/v${TERMUX_PKG_VERSION}.tar.gz"
+TERMUX_PKG_SHA256="1dd19f501a4f61b1370951efb76373ad29e9b5a76a6dae445fbfae2ed2648786"
+TERMUX_PKG_BUILD_DEPENDS="dotnet-sdk-8.0, aspnetcore-targeting-pack-8.0"
+TERMUX_PKG_DEPENDS="aspnetcore-runtime-8.0, ffmpeg, sqlite"
+TERMUX_PKG_BUILD_IN_SRC=true
+TERMUX_PKG_SERVICE_SCRIPT=(
+    "jellyfin"
+    "${TERMUX_PREFIX}/lib/dotnet/dotnet ${TERMUX_PREFIX}/share/jellyfin/Jellyfin.Server.dll --datadir ${TERMUX_ANDROID_HOME}/jellyfin"
+)
+
+termux_step_configure() {
+    dotnet8.0 restore
```

Comment:
Please make sure to use tabs for indentation as per the guidelines.
https://github.com/termux/termux-packages/wiki/Coding-guideline#script-formatting

The linter will mark this as an error and fail the CI run.
```suggestion
	dotnet8.0 restore
```